addons/binding/org.openhab.binding.blebox/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
I don't think this eclipse settings file should be added.

Reply:
You are right I have commit it by mistake. File removed.